test/integration/daemonset/daemonset_test.go

```diff
@@ -50,7 +50,8 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler"
 	"k8s.io/kubernetes/pkg/scheduler/algorithmprovider"
 	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
-	"k8s.io/kubernetes/pkg/scheduler/factory"
+	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
```

Comment:
nit: why not just `config` or just `schedulerconfig` ?

Reply:
I just followed the pattern in the codebase.

```
$ ag "kubeschedulerconfig \""
cmd/kube-scheduler/app/options/options.go
48:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"

cmd/kube-scheduler/app/options/insecure_serving.go
28:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"

cmd/kube-scheduler/app/options/deprecated.go
24:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"

cmd/kube-scheduler/app/options/configfile.go
25:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"

cmd/kube-scheduler/app/options/insecure_serving_test.go
28:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"

cmd/kube-scheduler/app/config/config.go
28:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"

cmd/kube-scheduler/app/server.go
48:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"

cmd/kube-scheduler/app/options/options_test.go
36:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"

test/integration/scheduler/scheduler_test.go
44:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"

pkg/scheduler/apis/config/scheme/scheme.go
23:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"

pkg/scheduler/scheduler_test.go
49:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"

pkg/scheduler/scheduler.go
40:	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
```

Reply:
In theory, aliases are only needed to resolve conflicts and for packages ending in `/v1` or similar. In this example, seeing `config.Foo` doesn't make it obvious that the config package is from "far away" in `pkg/scheduler`. I prefer `schedulerconfig` over `kubeschedulerconfig`.

Side note, when importing internal packages from your own code area, I don't think aliases are needed. But I see them all over the place in the scheduler code, e.g. `internalcache "k8s.io/kubernetes/pkg/scheduler/internal/cache"` :slightly_frowning_face:

Reply:
> Side note, when importing internal packages from your own code area, I don't think aliases are needed. But I see them all over the place in the scheduler code, e.g. `internalcache "k8s.io/kubernetes/pkg/scheduler/internal/cache"` 🙁

I'll do a follow-up PR for this. 😄